source/features/link-to-requested-change-comment.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '.';
+
+function init(): void {
+	observe('div.js-merge-message-container', {
+		constructor: HTMLDivElement,
+		add(messageContainer) {
+			for (const reviewItem of select.all('.merge-status-item.review-item', messageContainer)) {
```

Comment:
`observe` should select the final elements and should almost never contain a loop, since it can just call the `add` method on each element found.

The `constructor` is not necessary unless it adds require type information (e.g. `HTMLAnchorElement`)

```suggestion
	observe('merge-status-item.review-item', {
		add(reviewItem) {
```

Reply:
But this selection is too broad: `.review-item` even includes the whole dropdown and other links. I suggest:

1. changing this selector to select the dropdown link (the selector should end with `[href^="#pullrequestreview-"]`)
2. finding the container with something like `.closest('.review-status-item').querySelector('[title^="these changes"]')`
3. wrapping the specific "requested changes" text node, with `container.lastChild`

Reply:
I've updated my implementation similar to your suggestion. Please review it again.

Reply:
Seems good! Thanks